src/components/__tests__/CustomDropdown.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { createElement, ReactElement } from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { ListValue, EditableValue, ActionValue, ListAttributeValue, ValueStatus, ObjectItem } from "mendix";
+
+import CustomDropdown, { CustomDropdownComponentProps } from '../CustomDropdownComp';
+
+const createOption = () => ({
+    status: ValueStatus.Available,
+    items: [
+        { value: { displayValue: 'item1' } }
+    ]
+});
+
+const options = {
+    status: ValueStatus.Available,
+    items: [
+        { value: { displayValue: 'item1' } },
+        { value: { displayValue: 'item2' } },
+        { value: { displayValue: 'item3' } },
+    ]
+};
+
+const renderComponent = (override = {}) => {
+    const props = {
+        defaultValue: createOption(),
+        firstLabel: (item) => item.value,
+        secondLabel: (item) => item.value,
+        imgUrl: (item) => item.value,
```

Comment:
What is `item.value` in these three cases, that you initialize firstLabel, secondLabel and imgUrl to the same thing?

Reply:
It's the very beginning of the mock around the widget. All three will take the same and only value from the option provided to the widget. Of course the accessors has to be different to test whether accessors are being used.